src/components/course/routes/ExternalCourseEnrollment.jsx

```diff
@@ -89,7 +90,11 @@ const ExternalCourseEnrollment = () => {
           <Row>
             <Col>
               <h2 className="mb-3">
-                Your registration(s)
+                <FormattedMessage
+                  id="executive.education.external.course.enrollment.page.registration.title"
+                  defaultMessage="Your registration(s)"
```

Comment:
Do we need to handle singular vs plural here?

Reply:
> Do we need to handle singular vs plural here?

This text will be visible only when a user will come to this page while enrolling in a exec-ed course. So there is not any singular or plural scenario.